extensions/Worker.Extensions.ServiceBus/src/ServiceBusExtensionStartup.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Core;
+using Microsoft.Extensions.Azure;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+[assembly: WorkerExtensionStartup(typeof(ServiceBusExtensionStartup))]
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    public class ServiceBusExtensionStartup : WorkerExtensionStartup
+    {
+        public override void Configure(IFunctionsWorkerApplicationBuilder applicationBuilder)
+        {
+            if (applicationBuilder == null)
+            {
+                throw new ArgumentNullException(nameof(applicationBuilder));
+            }
+
+            applicationBuilder.Services.AddAzureClientsCore(); // Adds AzureComponentFactory
+
+            applicationBuilder.Services.Configure<WorkerOptions>((workerOption) =>
```

Comment:
With bypass deferred binding changes, we can remove these lines

```csharp
 applicationBuilder.Services.Configure<WorkerOptions>((workerOption) =>
            {
                workerOption.InputConverters.RegisterAt<ServiceBusReceivedMessageConverter>(0);
            });
```

Reply:
This seems to still be necessary. If I remove it, I get null parameter values.

Reply:
I gave this a try [here](https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=130528&view=results) and it worked. Let me know if you are still seeing issues with removing it.

Reply:
I was seeing the issue in my local E2E testing, not in any unit tests.

Reply:
This really shouldn't be required, I'm happy to also do some E2E testing on this tomorrow. If it's not working, this is a bug we should address with deferred binding

Reply:
Happy to help with the E2E testing! But will leave it in for now.

Reply:
Just cloned your branch this morning and did some manual E2E testing - all the samples for received message are working without this code block so I believe it's working as intended and it's safe to remove this code chunk

Reply:
I can repro the issue reliably if you want me to demo.

Reply:
Yup, this also works

<img width="875" alt="Screenshot 2023-05-19 at 10 17 50 AM" src="https://github.com/Azure/azure-functions-dotnet-worker/assets/2198905/13f5176c-e721-4f70-9707-e8b6b0a0ba0f">

Reply:
> I can repro the issue reliably if you want me to demo.

Yeah I'm curious what's different. Which version of the worker sdk are you using? It should be `1.9.0-preview1-20230512.2`